library/templates/dcp_censusblocks.yml

```diff
@@ -0,0 +1,36 @@
+dataset:
+  name: &name dcp_censusblocks
+  version: "{{ version }}"
+  acl: public-read
+
+  source:
+    url:
+      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nycb2010_{{ version }}.zip
+      subpath: nycb2010_{{ version }}/nycb2010.shp
+    geometry:
+      SRS: EPSG:4326
+      type: MULTIPOLYGON
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "STRING_EMPTY_AS_NULL=YES"
+
+  destination:
+    name: *name
+    geometry:
+      SRS: EPSG:4326
+      type: MULTIPOLYGON
+    options:
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+  info:
+    description: |
+      ### ESRI Shapefile
+      The Census Blocks for the 2010 US Census clipped to shoreline. These boundary files are derived
```

Comment:
Do we keep 2010 in here for now and manually change once we switch to 2020?

Reply:
I'll change 2010 for "current"

Reply:
@SPTKL what is your opinion on this?

Reply:
I think keeping 2010 is fine for now, but write a disclaimer somewhere in the description field? if say one day the link changed to 2020, this config should fail, and that's fine

Reply:
Is this disclaimer fine? 
"Source will be updated once the 2020 census data is available"